package.json

```diff
@@ -23,6 +23,9 @@
     "@angular/platform-browser": "11.2.5",
     "@angular/platform-browser-dynamic": "11.2.5",
     "@angular/router": "11.2.5",
+    "@types/chai": "^4.2.21",
```

Comment:
Why have all of these dependencies been added. As far as I can see you are not using them.